public/login.html

```diff
@@ -37,7 +37,7 @@ <h2>Login Form</h2>
           <button type="submit" class="btn btn-default">Login</button>
         </form>
         <br />
-        <p>Or sign up <a href="/">here</a></p>
+        <p>Or sign up <a href="styles/signup.html">here</a></p>
```

Comment:
Why is this changed?  It was correct going to the default location.  Please return it to the '/'...especially because the styles folder is for css and not html pages and the page doesn't exist there!